### PR TITLE
fixed RateLimiterUnion.consume return type

### PR DESCRIPTION
**Pull request by Omers-Frontegg**

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -354,7 +354,7 @@ export class RateLimiterMemcache extends RateLimiterStoreAbstract { }
 export class RateLimiterUnion {
     constructor(...limiters: RateLimiterAbstract[]);
 
-    consume(key: string | number, points?: number): Promise<RateLimiterRes[]>;
+    consume(key: string | number, points?: number): Promise<Record<string, RateLimiterRes>>;
```
Comment (animir) [inline lib/index.d.ts:357]: @Omers-Frontegg Hi, should it be Array of Record?

Comment (Omers-Frontegg) [reply lib/index.d.ts:357]: What do you mean by should?
The docs give an example with a return value of record 🙂

Comment (animir) [reply lib/index.d.ts:357]: @Omers-Frontegg Yes, that's right. Sorry.

**Comment by animir** [comment]
published in `v3.0.6`

**Pull request merged**